### ISSUE TITLE
Incompatibility with FoF Discussion Language

### ISSUE THREAD

**Issue by iamdarkle** [flamarkt/taxonomies#6]
I have started using taxonomies in a multi language community, the problem is that the taxonomy terms filter (either from the top of page filter or by clicking on a term from the discussion) does not seem to detect the selected language and therefore when filtering a term all discussions of that term are shown regardless of languages, it would be great if the filter could be applied correctly when having multiple languages.

**Comment by clarkwinkelmann** [comment]
I am not able to replicate. The filters seem to work as intended no matter in which order I apply them, on homepage, on tag page or during text search.

I have not tried on Flarum 1.5 yet.

```
Flarum core 1.4.0
PHP version: 8.0.22
MySQL version: 8.0.30-0ubuntu0.20.04.2
Loaded extensions: Core, date, libxml, openssl, pcre, zlib, filter, hash, json, pcntl, Reflection, SPL, session, standard, sodium, mysqlnd, PDO, xml, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, iconv, intl, exif, mysqli, pdo_mysql, pdo_sqlite, Phar, posix, readline, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xmlreader, xmlwriter, xsl, zip, Zend OPcache
+---------------------------+------------+------------------------------------------+
| Flarum Extensions         |            |                                          |
+---------------------------+------------+------------------------------------------+
| ID                        | Version    | Commit                                   |
+---------------------------+------------+------------------------------------------+
| flarum-flags              | v1.4.0     |                                          |
| flarum-tags               | v1.4.0     |                                          |
| flarum-suspend            | v1.4.0     |                                          |
| flarum-sticky             | v1.4.0     |                                          |
| flarum-lock               | v1.4.0     |                                          |
| flarum-approval           | v1.4.0     |                                          |
| flamarkt-backoffice       | 0.1.1      |                                          |
| migratetoflarum-fake-data | dev-master |                                          |
| kilowhat-audit-pro        | dev-master | 417ffcf5060fafc3e2c6d8d4e507f2c51413b1d9 |
| fof-discussion-language   | 1.2.7      |                                          |
| flarum-subscriptions      | v1.4.0     |                                          |
| flarum-mentions           | v1.4.0     |                                          |
| flarum-markdown           | v1.4.0     |                                          |
| flarum-lang-french        | v4.0.0     |                                          |
| flarum-lang-english       | v1.4.0     |                                          |
| flarum-bbcode             | v1.4.0     |                                          |
| flamarkt-taxonomies       | dev-main   | 55b0985d42b185dc76218f640f710a049aeddd59 |
+---------------------------+------------+------------------------------------------+
Base URL: http://1.4.flarum.localhost
Installation path: /home/clark/Projects/flarum-1.4
Queue driver: sync
Mail driver: smtp
Debug mode: ON
```

(commit `55b0985d42b185dc76218f640f710a049aeddd59` is Taxonomies 0.1.5)

**Comment by iamdarkle** [comment]
Yes, it's correct the filter works, but what happens is that it mixes the languages, for example here the language is in English but it shows both English and Spanish when applying a filter, this doesn't happen this way in 1.4? 

![Screenshot 2022-09-15 043845](https://user-images.githubusercontent.com/15818451/190300916-0b4fdfd2-c9da-4d85-94a2-57eabe24373a.png)


I'm testing it from 1.5 so it would be interesting if this does not happen in 1.4.

**Comment by clarkwinkelmann** [comment]
I'll try to test with 1.5 in the coming days.

I don't see the language selector in your screenshot. Are you using "Use detected language for default discussion filter" or "Only show discussions from the user's current locale on the Tags page" options? I haven't tried those.

**Comment by iamdarkle** [comment]
Here is the fof/discussion-language configuration I'm using:

![Screenshot 2022-09-15 185131](https://user-images.githubusercontent.com/15818451/190462688-9ac08f37-d21e-4862-8fcd-f117464d8ad7.png)

Yes, the selector doesn't appear because it is hidden via css, as with the language pack selector that also changes the filter is not so necessary at least in my case, but English was selected.

**Comment by clarkwinkelmann** [comment]
My next version of Taxonomies will improve the situation but the problem will probably only be fully fixed once https://github.com/FriendsOfFlarum/discussion-language/pull/32 is merged

**Comment by clarkwinkelmann** [comment]
Should be fully fixed with FoF Discussion Language 1.2.8

**Comment by iamdarkle** [comment]
Tested and working fine, thanks a lot!
